### DISCUSSION TITLE
New install on Ubuntu

### DISCUSSION THREAD

**Discussion by tcflint** [gregmac/Genmon-ESP32-Serial-Bridge#16] [General]
Hi,

Trying to do a setup with a VM on Synology.
I get the following when trying to install the software:

Copy configuration files to /etc/genmon/? (y/n)?y
Copying *.conf files to /etc/genmon/
mkdir: cannot create directory ‘/etc/genmon/’: File exists
Setup the raspberry pi onboard serial port? (y/n)?n
Not setting up serial port
Done.
Start genmon on boot? (y/n)?y
Updating crontab....
Crontab already contains genmon start script:
@reboot sleep 30 && /bin/bash /home/rob/genmon/startgenmon.sh start
rob@rob-Standard-PC-i440FX-PIIX-1996:~/genmon$

**Comment by gregmac** [comment]
Sorry, that's not part of this repository.

Not even sure what your question is exactly, it kind of just looks like you ran the install twice.
